### ISSUE TITLE
[P1][Sourcing][Accessibility]Table: The empty cell information is not conveyed through screenreader

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10320]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
This issue is in continuation of https://github.com/SAP/fundamental-ngx/issues/9964.
Expectation-
 to meet UI5 table or as per w3 standards.
The "Empty" cell needs to be communicated to disabled users via screen readers.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15

#### If this is a bug, please provide steps for reproducing it.
can be replicable on NGX website : https://sap.github.io/fundamental-ngx/#/platform/table/basic
#### Is there anything else we should know?
Reported by customer (high priority)

**Comment by nikolay-kolarov** [comment]
Hi,

Unfortunately, this does not come for free from the HTML and in UI5 and UI5 WebComponents there is a custom string added to the aria-labelledby, when a given cell is empty.

Best Regards,
Nikolay

**Comment by droshev** [comment]
fixed with `0.43.8`

**Comment by sagar7259** [comment]
Hello,

As per the retest activity this issue is still not fixed. On accessing the empty cells of the table its corresponding "Empty" information is not read.

Speech history:
"**Description**
1 of 5
Column 3
Column 3 of 5, row: 2 of 9
**Price**
1 of 5
Column 4
Column 4 of 5, row: 2 of 9
**Status**
1 of 5
Column 5
Column 5 of 5, row: 2 of 9"

Here 'Description', 'Price' and 'Status' columns have empty cells, but this information is not read to the users. 
The fix should be part of all the tables across NGX and with respect to any one table.

@rvamudha could you please reopen this ticket.?

Regards,
Sagar
